x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/downsample/DownsampleActionValidationException.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+package org.elasticsearch.xpack.core.downsample;
+
+import org.elasticsearch.action.ActionRequestValidationException;
+
+public class DownsampleActionValidationException extends ActionRequestValidationException {}
```

Comment:
I don't think this exception gets serialised over the wire. But the same applied for `RollupActionRequestValidationException`. Can we reuse an exception that does have serialization support? (which I think is any exception mentioned in `ElasticsearchException#CLASS_TO_ELASTICSEARCH_EXCEPTION_HANDLE` map)

Reply:
I removed `DownsampleActionValidationException` and replaced it with the ubiquitous `ActionRequestValidationException`. There seems to be little gains in having a subclass.

Also, all `ElastisearchException` subclasses are used for internal errors and not for request validations. 
Not sure how serializing is handled, but class `ActionRequestValidationException` seems to be
all over the place for action request validation errors.

Commit [e1b22a2](https://github.com/elastic/elasticsearch/pull/90295/commits/e1b22a23da2d09e14a33828a4cf3d48708f2d5a7)

Reply:
I think rely on the fact that only the message gets serialised. We always return a 400 error, because `ActionRequestValidationException` extends from IllegalArgumentException. 
I think this good enough in this case, since the `validationErrors` are being added to the exception message and there are not other fields that need to be serialised.